px-context-browser-trigger.html

```diff
@@ -0,0 +1,82 @@
+<link rel="import" href="../px-icon-set/px-icon-set.html"/>
+<link rel="import" href="../px-icon-set/px-icon.html"/>
+
+<dom-module id="px-context-browser-trigger">
+  <template>
+    <style>
+      :host {
+        display: inline-flex;
+        height: var(--px-context-browser-trigger-height, 32px);
+        width: var(--px-context-browser-trigger-width, 32px);
+        --iron-icon-width: 100%;
+        --iron-icon-height: 100%;
+        padding: 2px;
+        box-sizing: border-box;
+        cursor: pointer;
+        align-items: center;
+        justify-content: center;
+      }
+      :host(:hover) {
+        background-color: var(--px-context-browser-trigger-hover-background, lightgray);
+      }
+      px-icon {
+        stroke: var(--px-context-browser-trigger-icon-color, currentcolor);
+      }
+    </style>
+    <px-icon icon="[[_getIcon(favorites)]]"></px-icon>
+  </template>
+  <script>
+    Polymer({
+      is: 'px-context-browser-trigger',
+
+      properties: {
+        /**
+         * Read-only reference to the trigger element. Data-bind this property
+         * into the px-context-browser `openTrigger` or `favoritesTrigger`
+         * property to open the context browser when this icon is tapped.
+         */
+        trigger: {
+          type: HTMLElement,
+          readOnly: true,
+          notify: true,
+          value: null
+        },
+
+        /**
+         * Enable to use this trigger to open the px-context-browser favorites.
+         * If enabled, bind this element's `trigger` property to the context
+         * browser's `favoritesTrigger` property.
+         */
+        favorites: {
+          type: Boolean,
+          value: false
+        }
+      },
+
+
+      attached() {
+        this._setTrigger(this);
+      },
+
+      detached() {
+        this._setTrigger(null);
+      },
+
+      listeners: {
+        'tap' : '_handleTapped'
+      },
+
+      _handleTapped(evt) {
+        evt.preventDefault();
```

Comment:
Do we need to normalise event? Maybe not given simple usage. Just worth checking...